resources/shaders/glbillbshader.frag

```diff
@@ -17,7 +17,7 @@ struct FogParam {
 
 uniform sampler2D texture0;
 uniform usamplerBuffer palbuf;
-uniform bool repaint;
+uniform bool repaint; // TODO(captainurist): always true?
```

Comment:
Yes. Can drop and just use the fact that billboards with non zero paletteIds will need repainting I think.

Reply:
OK, will do this separately.